src/componants/Header.jsx

```diff
@@ -37,15 +37,19 @@ function Header() {
 					{categories.map((Acategory) => {
 						if (category === Acategory.slug) {
 							return (
-								<button className="highlightedcatButton" key={Acategory.slug}>
+                <Link to={`/${Acategory.slug}`} key={Acategory.slug}>
+								<button className="highlightedcatButton" >
 									{Acategory.slug}
 								</button>
+                </Link>
```

Comment:
This works really well and I love the red highlight. I would also make a button for all articles so users can go back to all of them after they have clicked a topic. Currently, users have to refresh or click your header to do this